backend/src/main/kotlin/app/ehrenamtskarte/backend/application/database/repos/EakApplicationRepository.kt

```diff
@@ -75,17 +75,18 @@ object EakApplicationRepository {
 
     fun getApplications(regionId: Int): List<ApplicationView> {
         return transaction {
-            EakApplicationEntity.find { EakApplications.regiondId eq regionId }
+            EakApplicationEntity.find { EakApplications.regionId eq regionId }
                 .map { ApplicationView(it.id.value, it.regionId.value, it.createdDate.toString(), it.jsonValue) }
         }
     }
 
-    fun delete(applicationId: Int): Boolean {
+    fun delete(applicationId: Int, graphQLContext: GraphQLContext): Boolean {
         return transaction {
             val application = EakApplicationEntity.findById(applicationId)
                 if (application != null) {
+                    val applicationDirectory = File(graphQLContext.applicationData, application.id.toString())
                     application.delete()
```

Comment:
Is it possible that this throws an exception and the next line is not executed?

Reply:
I guess, this is not reaally a problem... If the above throws an exception, the frontend also communicates taht there was a problem deleting the application and hence will still be present in the list of applications.
Then, they'd have to call an administrator anyway why this application cannot be deleted...